lib/tasks/book2.rake

```diff
@@ -83,12 +83,12 @@ def genbook(code, &get_content)
   # revert internal links decorations for ebooks
   content.gsub!(/<<.*?\#(.*?)>>/, "<<\\1>>")
 
-  asciidoc = Asciidoctor::Document.new(content, template_dir: template_dir, attributes: { "lang" => code})
+  asciidoc = Asciidoctor::Document.new(content, attributes: { "lang" => code})
   html = asciidoc.render
   alldoc = Nokogiri::HTML(html)
   number = 1
 
-  Book.destroy_all(edition: 2, code: code)
+  Book.where(edition: 2, code: code).destroy_all
```

Comment:
Heh, this is undoing commit 313dfb7d1890f5f9e52e7e6f51a268f20810f236, which came from a suggestion in https://github.com/git/git-scm.com/pull/1225#discussion_r202065311. I blame @ttaylorr :wink: